packages/terminal/src/browser/base/terminal-widget.ts

```diff
@@ -166,6 +166,11 @@ export abstract class TerminalWidget extends BaseWidget {
     abstract waitOnExit(waitOnExit?: boolean | string): void;
 }
 
+export interface TerminalIcon {
```

Comment:
Isn't this a `ThemeIcon`?

Reply:
There are some other `ThemeIcon` types I found.
But you mean the one from common/theme.ts? I think we can re-use it.

Reply:
I removed the new interface definition, thank you for the hint!